src/components/HTMLEngineProvider/index.native.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
 import BaseHTMLEngineProvider from './BaseHTMLEngineProvider';
-import {defaultProps, propTypes} from './htmlEnginePropTypes';
+import HTMLEngineProviderProps from './types';
 
-function HTMLEngineProvider(props) {
+function HTMLEngineProvider({debug = false, children = null}: HTMLEngineProviderProps) {
```

Comment:
I think we can remove default `null` value for children, wdyt?

Reply:
I agree, removing it 👍